src/controllers/events.controller.js

```diff
@@ -255,4 +263,32 @@ exports.status = async (req, res, next) => {
     return res.status(422).json(responseMsg.error(errorMsg.params.EVENTID,
         errorMsg.messages.EVENT_NOT_FOUND));
   }
-};
\ No newline at end of file
+};
+
+// upload event audio to s3
+exports.upload = async (req, res, next) => {
+  const filename = req.files.file.name;
```

Comment:
do you have to check whether only 1 file is uploaded?

Reply:
and, do you want to change the filename? since different users may upload files with a same filename